### PR TITLE
tests: cleanup extensive environment variables

### PR DESCRIPTION
**Pull request by zonggen**
Signed-off-by: Allen Bai <abai@redhat.com>

### CODE REVIEW

File: tests/functional/utils.py [hunk 1]
```diff
@@ -43,7 +49,7 @@ def get_run_result(secrets, run_id):
     return run['conclusion']
 
 
-@retry(stop_max_delay=15_000)
+@retry(stop_max_delay=15_000, wait_fixed=1000)
```
Comment (zonggen) [inline tests/functional/utils.py:52]: Hit GitHub request limit while testing, set this interval period to slow down the requests.

File: tests/functional/notifier.py [hunk 2]
```diff
@@ -94,6 +92,9 @@ def create_verification_issue(chart_name, chart_owners, report_url, software_nam
     access_token -- An optional github access token secret. If not passed will try to get from GITHUB_AUTH_TOKEN environment variable\n
     """
 
+    if os.environ.get('NOTIFY_ID'):
```
Comment (zonggen) [inline tests/functional/notifier.py:95]: Use this `NOTIFY_ID` if provided to override the chart_owners to tag

**Review by dperaza4dustbit — APPROVED** [review]
I think we can improve these test module to reduce the duplicate logic like:
if not bot_token:
        bot_name = "github-actions[bot]"
        ...

File: tests/functional/notifier.py (hunk 2 above)
Comment (mmulholla) [reply tests/functional/notifier.py:95]: The notify id could be "OWNERS", "none" or a githib id. I would expect to see some logic depending in its value, like reading the OWNERS file if set to OWNERS. Also if "none" there is no id to notify.

**Comment by mmulholla** [comment]
Also I get the impression there is a lot of duplicated code. Need to look into making some common functions.

**Pull request merged**